src/hssm/hssm.py

```diff
@@ -2048,9 +2075,12 @@ def __jitter_initvals_all(self, jitter_epsilon: float) -> None:
             starting_value_tmp = starting_value + np.random.uniform(
                 -jitter_epsilon, jitter_epsilon, starting_value.shape
             ).astype(np.float32)
-            self.pymc_model.set_initval(
-                self.pymc_model.named_vars[name_tmp], starting_value_tmp
-            )
+
+            if not isinstance(self._initvals, dict):
+                raise ValueError("self._initvals should not be a dictionary.")
+            else:
```

Comment:
`self._initvals` can only be of type `dict` right? It's initialized as a `dict`

Reply:
most of these are just in response to mypy bugging me endlessly :D
will try to do a round of improvements. @digicosmos86